assets/js/leaflet/leafembed.js

```diff
@@ -1,28 +1,17 @@
-
-
 var map;
-var ajaxRequest;
-var plotlist;
 var plotlayers=[];
+var iconsList={'qso':{'color':'#FF0000','icon':'fas fa-dot-circle'}};
 
-var greenIcon = L.icon({
-    iconUrl: icon_dot_url,
-    iconSize:     [10, 10], // size of the icon
-});
+var stationIcon = L.divIcon({className:'cspot_station'});
+var qsoIcon = L.divIcon({className:'cspot_qso'}); //default (fas fa-dot-circle red)
+var qsoconfirmIcon = L.divIcon({className:'cspot_qsoconfirm'});
+var redIconImg = L.icon({ iconUrl: icon_dot_url, iconSize: [10, 10] }); // old //
 
 var osmUrl = $('#leafembed').attr("tileUrl");
 
-function initmap(ShowGrid = 'No', MapTag = 'map') {
-    // set up AJAX request
-    ajaxRequest=getXmlHttpObject();
-    if (ajaxRequest==null) {
-        alert ("This browser does not support HTTP Request");
-        return;
-    }
```

Comment:
Don't know if that is a good idea to remove this.
I think it was there for a reason. It checked if Browser was/is up2date and throws an error if the browser doesn't support ajax-requests.